code/modules/projectiles/guns/energy/magic.dm

```diff
@@ -21,6 +21,11 @@
 /obj/item/weapon/gun/energy/staff/emp_act(severity) // to prevent it from breaking.
 	return
 
+/obj/item/weapon/gun/energy/staff/pickup(mob/living/user as mob)
+	if(!user.is_wizard())
+		user << "<span class='warning'>When you pick up \the [src], you can feel it humming unsettlingly through your limbs. You probably shouldn't try to use this...</span>"

```

Comment:
Remove the last bit. It unnecessarily breaks the 4th wall and the meaning of it is already conveyed by the rest of the text _and_ dizziness.